pkg/cmd/delete/root.go

```diff
@@ -0,0 +1,53 @@
+package delete
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/cnoe-io/idpbuilder/pkg/kind"
+	"github.com/spf13/cobra"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+var (
+	// Flags
+	buildName string
+)
+
+var DeleteCmd = &cobra.Command{
+	Use:   "delete",
+	Short: "Delete an IDP cluster",
+	Long:  ``,
+	RunE:  delete,
+}
+
+func init() {
+	DeleteCmd.PersistentFlags().StringVar(&buildName, "buildName", "localdev", "Name for build (Prefix for kind cluster name, pod names, etc).")
```

Comment:
The text "Name for build (Prefix for kind cluster name, pod names, etc)." is confusing. 
I suggest "Name of the kind cluster to be deleted."